contrib/builder/arm/deb/debian-jessie/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ioft/armhf-debian:jessie
```

Comment:
@thaJeztah Couple of weeks ago I've been asking @jfrazelle  if it would make sense to a least move all the necessary images for building and testing Docker on ARM to the official Docker image library. Back then she said that for the time being we should just use the private images. 

I guess @jfrazelle meant that we should clean up the images later as a follow up task. 

In my opinion this is really about creating an official Docker image library for ARM, which I really would like to see sooner than later. Still I am not sure if this should be something we solve as part of this PR.